packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -54,19 +59,39 @@ export class OdsTooltip {
     this.triggerElement?.addEventListener('blur', this.boundHide);
     this.triggerElement?.addEventListener('focus', this.boundShow);
     this.triggerElement?.addEventListener('mouseenter', this.boundShow);
-    this.triggerElement?.addEventListener('mouseleave', this.boundHide);
+    this.triggerElement?.addEventListener('mouseleave', () => setTimeout(this.boundHide, 50));
+    this.el.addEventListener('mouseenter', () => {
+      this.isTooltipHover = true;
+      return this.boundShow();
+    });
+    this.el.addEventListener('mouseleave', () => {
+      this.isTooltipHover = false ;
+      return this.boundHide();
+    });
+
   }
 
   disconnectedCallback() : void {
     this.triggerElement?.removeEventListener('blur', this.boundHide);
     this.triggerElement?.removeEventListener('focus', this.boundShow);
     this.triggerElement?.removeEventListener('mouseenter', this.boundShow);
     this.triggerElement?.removeEventListener('mouseleave', this.boundHide);
+    this.el.removeEventListener('mouseenter', () => {
+      this.isTooltipHover = true;
+      return this.boundShow();
+    });
+    this.el.removeEventListener('mouseleave', () => {
+      this.isTooltipHover = false ;
+      return this.boundHide();
+    });
   }
 
   render(): FunctionalComponent {
     return (
-      <Host class={ `ods-tooltip ods-tooltip--${this.strategy}` }>
+      <Host
+        id={ this.tooltipId }
```

Comment:
why ?
`id` is a native attribute that can be added to the web-component directly by the integrator.